<launch>
   <arg name="vel_linear" value="0.3"/>
   <arg name="vel_angular" value="1"/>
   <arg name="pub_rate" value="20"/>

   <node pkg="joy" type="joy_node" name="joystick" respawn="true">
       <param name="dev" value="/dev/input/js0" type="string"/>
   </node>

   <node pkg="bulldog_teleop" type="joystick_control" name="joystick_control" output="screen"> 
       <param name="vel_linear"   value="$(arg vel_linear)"   type="double"/>
       <param name="vel_angular"  value="$(arg vel_angular)" type="double"/>
       <param name="pub_rate"  value="$(arg pub_rate)" type="int"/>
   </node>
</launch>